Analysis/Tasks/PWGHF/HFJpsiToEECandidateSelector.cxx

```diff
@@ -79,9 +79,10 @@ struct HFJpsiToEECandidateSelector {
     if (track.charge() == 0) {
       return false;
     }
```

Comment:
Please remove this check.

Reply:
I will do that, I also commented everything that has to do with the TPC, do you want me to uncomment this before committing?